.gitignore

```diff
@@ -44,3 +44,6 @@ spec/examples.txt
 
 # Brew files
 Brewfile.lock.json
+
+# kibana folder. asdf does not support kibana 8 so you will have to download it by hand
+kibana-*/*
```

Comment:
Instead of adding this to the repo's `.gitignore`, I would suggest putting this line in your `.git/info/exclude` file instead, which is like your own personal `.gitignore` file.

(You could also move your kibana setup to someplace like `/usr/local` if you might want to use it for other projects).